index.html

```diff
@@ -1,8 +1,196 @@
+                    <p>Pubs were also a big part of my morning routine. There was a lovely pub (pictured, above) called the Quays that served the most delicious Irish stew, along with fresh seafood that was caught fresh that morning.</p>
+                    <footer><a href="#top">Back to Top</a></footer>
+                    
+                <h3>Tours</h3>
+                    <h4>Out of Dublin, Ireland</h4>
+
+                    <img src="images/trinity.jpg" alt="A photo of the entrance to Trinity College in Dublin, depicting an old tower to the left with trees and grass to the right, with buildings in the background." width = "750" height="420">
+                    <img src="images/library.jpg" alt="A photo of the Trinity College Library, showing countless rows of bookshelves on multiple floors visible and large domed ceilings." width = "750" height="420">
+                    <p>Trinity Library was beautiful to visit, the architecture is amazing. I loved the library, I wish I had one like it.</p>
+                    <p></p>
+                    <img src="images/tara.jpg" alt="An ancient standing stone in a green field, surrounded by a panoramic view of a great deal of Ireland in the distance." width="750" height="420">
+                    <p>The hill of Tara was for many years where the high kings of Ireland were crowned. The iconic view was one reason they believe the spot was chosen. You can see a large portion of Ireland from the spot. The standing stone, called 'The Stone of Destiny' was the place high kings ascended to the throne, for centuries.  </p>
+                    <img src="images/tara2.jpg" alt="" width="750" height="420">
+                    <p>While I was on a tour of Tara, a woman tripped and fell and broke her leg. After that happened, I mostly remembered that part. But Tara itself was very beautiful.</p>
+                    <footer><a href="#top">Back to Top</a></footer>
+
+                    <h4>Out of Galway, Ireland</h4>
+                    <img src="images/cliffs.jpg" alt="The cliffs of Moher. Green towering cliffsides on the bottom left, the sea and a colorful sunset on the right." width="750" height="420">
+                    <p>The cliffs of Moher are beautiful, and slightly scary. It was very windy the day I went, with sudden gusts that kind of pushed you around a little.</p>
+                    <p>The view was amazing, and they had a lovely historical tour as well. They had an extensive gift shop and a food court area that served some lovely ice cream.</p>
+                    <footer><a href="#top">Back to Top</a></footer>
+
+                <h3>Historical Sites</h3>
+                    <img src="images/newgrange.jpg" alt="An ancient passage with neolithic carvings depicting spirals and chevrons." width = "750" height="420">
+                    <p>I had the privilege to see some of mankind's earliest art at newgrange. It's a passage tomb sacred to neolithic druids.</p>
```

Comment:
Your indenting is [*chef's kiss*](https://i.giphy.com/media/3o7qDWp7hxhi1N8oF2/giphy.webp)